src/App.tsx

```diff
@@ -1,24 +1,206 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import React, {
+  useState, useEffect, useMemo, useRef,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+import { getTodos, patchTodos, postTodos } from './api/todos';
+import { ShowState } from './types/ShowState';
+import { ErrorTypes } from './types/ErrorTypes';
+import { Error } from './components/Error';
+import { Loader } from './components/Loader';
+
+import { USER_ID } from './utils/userId';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | []>([]);
+  const [showState, setShowState] = useState<ShowState>(ShowState.All);
+  const [error, setError] = useState<ErrorTypes | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isLoading, setIsLoading] = useState(false);
```